user_data/hyperopts/sample_hyperopt.py

```diff
@@ -6,10 +6,11 @@
 from functools import reduce
 
 import numpy
-from skopt.space import Categorical, Dimension, Integer, Real
 
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 from freqtrade.optimize.hyperopt_interface import IHyperOpt
+from freqtrade.optimize.hyperopt_interface import Categorical, Dimension, Integer, Real
```

Comment:
I don't think this is currently necessary (and it's problematic for another reason too):

a) it works without this import from hyperopt_interface - also witht he below command
b) since this is a file we expect users to copy / keep / modify, it's a user interface - so if we add this now - we are kind of constraint to keep these imports in the interface (or throw deprecation warnings ourselfs). This is not per se problematic - but odd if you need to investigate how these actually work, as it's not clear it's passed through from skopt - so users may google the wrong stuff.

```bash 
freqtrade --config config.json.example --datadir freqtrade/tests/testdata hyperopt -e 5 --customhyperopt SampleHyperOpts
```